lib/gears/screen.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------------------------------------
+-- @author Uli Schlachter
+-- @copyright 2016 Uli Schlachter
+-- @release @AWESOME_VERSION@
+-- @classmod gears.screen
+---------------------------------------------------------------------------
+
+local screen = screen
+
+local module = {}
+
+--- Call a function for each existing and created-in-the-future screen.
+-- @tparam function func The function to call.
+function module.connect_for_each_screen(func)
```

Comment:
The plan is to not define any screens before loading the config, so yeah, your edit explains the reason for this. :-)
TODO: Merge this into awful.screen if no users in gears/wibox show up.